### PR TITLE
main/libspelling: new package (0.2.0)

### PR DESCRIPTION
**Pull request by GeopJr**
Not sure if this should be in main instead

### CODE REVIEW

**Comment by q66** [comment]
does anything use it?

**Comment by GeopJr** [comment]
I believe [gtranslator](https://gitlab.gnome.org/GNOME/gtranslator/) does and I use it for Tuba. It's the only spelling checker for GTK4 so I assume more apps will do too :shrug:

**Comment by nekopsykose** [comment]
can probably just move it to main when that happens since that takes a few seconds anyway

**Comment by GeopJr** [comment]
FWIW, gnome-text-editor also plans to switch to it eventually [issue](https://gitlab.gnome.org/GNOME/gnome-text-editor/-/issues/594) / [MR](https://gitlab.gnome.org/GNOME/gnome-text-editor/-/merge_requests/117). Moving it to main then sounds good to me!

**Comment by q66** [comment]
in that case, why not just put it in main straight away?

**Comment by GeopJr** [comment]
Moved to main

File: main/libspelling/template.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "vala",
+    "glib-devel",
+]
+makedepends = [
+    "enchant-devel",
+    "glib-devel",
+    "gtk4-devel",
+    "gtksourceview-devel",
+    "icu-devel",
+    "vala",
```
Comment (q66) [inline main/libspelling/template.py:19]: shouldn't need vala in makedepends

**Pull request merged**